### PR TITLE
Update Add-ADGroupMember.md

### PR DESCRIPTION
**Pull request by HerbertMauerer**
As a result of windows bug "24359250: [RS1] Using Permissive Modify causes extra event 5136 to be written for Group membership changes" a new parameter was added to this cmdlet. It already appears in the online help in Powershell ISE.

### CODE REVIEW

**Comment by HerbertMauerer** [comment]
Hello,

I think this is stuck. Is there a ETA of this being released?
I have a customer waiting for it.

Herbert Mauerer
[cid:image001.png@01D6B99E.A4E54AF0]<https://campus.partners.extranet.microsoft.com/sites/campus/Pages/wwcommunities.aspx>
Senior Escalation Engineer
Customer Services and Support EMEA
WW Communities Debugging<http://aka.ms/wwcomdebug>, SME
Which WW Community should you join? Click here<https://microsoft.sharepoint.com/teams/campusWWCs/Pages/Landing.aspx>
[cid:image002.jpg@01D6B99E.A4E54AF0]
Do you have feedback for me? Please ask me for a connect survey.

**Comment by e0i** [comment]
Closing and re-opening to refresh validation checks.

**Comment by e0i** [comment]
Closing and re-opening to run the validation checks.

**Comment by e0i** [comment]
cc @JohanFreelancer9 for copy editing review.

**Review by get-itips — CHANGES_REQUESTED** [review]
The file was updated after this PR was created so it is currently resulting in a merge conflict.
Please review the conflicts, thanks

File: docset/windows/addsadministration/Add-ADGroupMember.md [hunk 1]
```diff
@@ -329,6 +329,25 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisablePermissiveModify
+Group membership updates use permissive modify by default. This suppresses an error when adding a member that is already member of the group.
```
Comment (JohanFreelancer9) [inline docset/windows/addsadministration/Add-ADGroupMember.md:333]:
```suggestion
Group membership updates use permissive modify by default. This suppresses an error when adding a member that is already a member of the group.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

File: docset/windows/addsadministration/Add-ADGroupMember.md (hunk 1 above)
Comment (HerbertMauerer) [reply docset/windows/addsadministration/Add-ADGroupMember.md:333]: Please excuse me, this is one of my first and until now biggest changes I make in github directly. I made a few mistakes, and I have created new pull requests and the one where I seem to have gotten things right are 2228 and 2229.

So I think we should cancel this one. You can check the other PRs and edit review them. Thank you.

Herbert Mauerer

**Review by tseward — APPROVED** [review]

**Comment by HerbertMauerer** [comment]
I am polling the topics and they are still not updated. is there an ETA for the changes to publish?

File: docset/windows/addsadministration/Add-ADGroupMember.md (hunk 1 above)
Comment (get-itips) [reply docset/windows/addsadministration/Add-ADGroupMember.md:333]: @e0i the author is asking here to close this PR as he created others to replace it, and also, this one has conflict merges.

**Comment by e0i** [comment]
Closing because of the validation errors.

**Pull request closed without merging**